src/coreclr/src/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/FieldAccessors/StaticFieldAccessor.cs

```diff
@@ -66,5 +66,6 @@ public sealed override int Offset
         protected abstract object GetFieldBypassCctor();
         protected abstract void SetFieldBypassCctor(object value, BinderBundle binderBundle);
         protected abstract void SetFieldDirectBypassCctor(object value);
+        protected abstract bool IsFieldInitOnly { get; }
```

Comment:
Looks like we don't need this here - maybe introduce the abstract method on the WritableStaticFieldAccessor so that we don't need to make changes in this file and in the literal accessor?